test/unit-complete/.terraform.lock.hcl

```diff
@@ -0,0 +1,21 @@
+# This file is maintained automatically by "terraform init".
```

Comment:
is this file needed to be committed?

Reply:
if not, check `unit-disabled/` as well.

Reply:
you're right, it shouldn't be part of this PR - will change

Reply:
removed in 985e154d